jenkins/tenant/aws/qe/qe_02/PolicyJenkinsfileDaily

```diff
@@ -0,0 +1,91 @@
+accounts_list = ['konflux-fedora': 'dbaez@redhat.com, eedri@redhat.com, ilario@redhat.com, xjiang@redhat.com, ebattat@redhat.com, maniskum@redhat.com',
```

Comment:
Why is it a list and not dictionary of list ?
{'konflux-fedora': ['dbaez@redhat.com, eedri@redhat.com, ilario@redhat.com, xjiang@redhat.com, ebattat@redhat.com, maniskum@redhat.com']}

Reply:
jenkins supports dict like this.

Reply:
Ok got u